.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
```

Comment:
Unrelated to this pr, but I noticed the license file on master has `Copyright [yyyy] [name of copyright owner]`. It's good to fix it as well